gno.land/cmd/gnoland/main.go

```diff
@@ -116,6 +124,10 @@ func exec(c *gnolandCfg) error {
 		cfg.Consensus.CreateEmptyBlocksInterval = 60 * time.Second
 	})
 
+	if c.rpcListenAddr != "tcp://127.0.0.1:26657" {
+		cfg.RPC.ListenAddress = c.rpcListenAddr
+	}
```

Comment:
This is not the way to manage this. Instead, you should parse flags directly against the unmarshalled config file.